### PR TITLE
fix: CLI help for PayForBlob

### PR DESCRIPTION
**Pull request by rootulp**
Closes https://github.com/celestiaorg/celestia-app/issues/2990

## Testing

```shell
$ ./build/celestia-appd tx blob PayForBlob --help
Pay for data blob(s) to be published to Celestia.
To publish a single blob, specify the namespaceID and blob via CLI arguments.
To publish multiple blobs, use the --input-file flag with the path to a JSON file.
The JSON should look like:

{
	"Blobs": [
		{
			"namespaceID": "0x00010203040506070809",
			"blob": "0x48656c6c6f2c20576f726c6421"
		},
		{
			"namespaceID": "0x00010203040506070809",
			"blob": "0x48656c6c6f2c20576f726c6421"
		}
	]
}

The namespaceID is the user-specifiable portion of a version 0 namespace.
The namespaceID must be a hex encoded string of 10 bytes.
The blob must be a hex encoded string of non-zero length.

Usage:
  celestia-appd tx blob PayForBlobs [namespaceID blob] [flags]
```

### CODE REVIEW

**Review by cmwaters — APPROVED** [review]

**Review by evan-forbes — APPROVED** [review]

File: x/blob/client/cli/payforblob.go [hunk 1]
```diff
@@ -52,27 +54,28 @@ func CmdPayForBlob() *cobra.Command {
...
+			"blob": "0x48656c6c6f2c20576f726c6421"
 		}
+	]
+}
 
-		namespaceID is the user-specifiable portion of a version 0 namespace. It must be a hex encoded string of 10 bytes.\n
-		blob must be a hex encoded string of any length.\n
+The namespaceID is the user-specifiable portion of a version 0 namespace.
+The namespaceID must be a hex encoded string of 10 bytes.
+The blob must be a hex encoded string of any length.
```
Comment (staheri14) [inline x/blob/client/cli/payforblob.go:78]: [Question] Is zero-length blob an acceptable input? if not, then it is worth mentioning that blobs should have non-zero length.

File: x/blob/client/cli/payforblob.go [hunk 2]
```diff
@@ -81,17 +84,16 @@ func CmdPayForBlob() *cobra.Command {
 				return err
 			}
 
-			// If there is a file path input we'll check for the file extension
 			if path != "" {
-				if filepath.Ext(path) != ".json" {
-					return fmt.Errorf("invalid file extension, require json got %s", filepath.Ext(path))
+				if filepath.Ext(path) != FileInputExtension {
```
Comment (staheri14) [inline x/blob/client/cli/payforblob.go:88]: [Question] Is there a check in the code for whether the file exists in the specified path (hence showing proper error if not)?

**Review by staheri14 — DISMISSED** [review]
LGTM!

File: x/blob/client/cli/payforblob.go (hunk 1 above)
Comment (rootulp) [reply x/blob/client/cli/payforblob.go:78]: Agreed, good call!

Comment (rootulp) [reply x/blob/client/cli/payforblob.go]: 2c37b373a4152e2d67faeb8d0725dbdc8e1a4444

**Review by staheri14 — APPROVED** [review]

File: x/blob/client/cli/payforblob.go (hunk 2 above)
Comment (rootulp) [reply x/blob/client/cli/payforblob.go:88]: Looks like yes.
```
$ ./build/celestia-appd tx blob pay-for-blob --input-file foo.json --from validator
Error: open foo.json: no such file or directory
```

**Pull request merged**